Firestore/core/src/bundle/bundle_reader.cc

```diff
@@ -46,12 +48,12 @@ BundleMetadata BundleReader::GetBundleMetadata() {
   }
 
   std::unique_ptr<BundleElement> element = ReadNextElement();
-  if (!element || element->ElementType() != BundleElementType::Metadata) {
+  metadata_loaded_ = true;
```

Comment:
The reason is this is a mistake..

Although it does not matter much because once it is failed, clients should not interact with reader again.

Fixed it still.